osmdroid-android/src/main/java/org/osmdroid/tileprovider/cachemanager/CacheManager.java

```diff
@@ -61,24 +63,31 @@
  * @author M.Kergall
  * @author Alex
  * @author 2ndGAB
+ * @author F.Fontaine
  */
 public class CacheManager {
 
     protected final MapTileProviderBase mTileProvider;
     protected final IFilesystemCache mTileWriter;
-    protected final MapView mMapView;
+    protected final int mMinZoomLevel;
+    protected final int mMaxZoomLevel;
     protected Set<CacheManagerTask> mPendingTasks = new HashSet<>();
 
     public CacheManager(final MapView mapView) {
-        mTileProvider = mapView.getTileProvider();
-        mTileWriter = mapView.getTileProvider().getTileWriter();
-        mMapView = mapView;
+        this(mapView, mapView.getTileProvider().getTileWriter());
     }
 
     public CacheManager(final MapView mapView, IFilesystemCache writer) {
-        mTileProvider = mapView.getTileProvider();
-        mTileWriter = writer;
-        mMapView = mapView;
+        this(mapView.getTileProvider(), writer, mapView.getMinZoomLevel(), mapView.getMaxZoomLevel());
+    }
+
+    public CacheManager(final MapTileProviderBase pTileProvider,
```

Comment:
since this is a new constructor, please add a javadoc comment that references the issue for the PR and an @ since 5.6.5